api/controllers/CampaignBotController.js

```diff
@@ -462,50 +481,59 @@ class CampaignBotController {
 
     const dateSubmitted = Date.now();
     const submission = req.signup.draft_reportback_submission;
-
     submission.submitted_at = dateSubmitted;
+
     return submission
       .save()
       .then(() => {
         this.debug(req, `updated submission:${submission._id.toString()}`);
 
-        /* eslint-disable no-param-reassign */
-        req.signup.reportback = rbid;
-        req.signup.total_quantity_submitted = req.signup.draft_reportback_submission.quantity;
-        req.signup.updated_at = dateSubmitted;
-        req.signup.draft_reportback_submission = undefined;
-        /* eslint-enable no-param-reassign */
-        req.signup.save();
+        const signup = req.signup;
+        signup.reportback = rbid;
+        signup.total_quantity_submitted = req.signup.draft_reportback_submission.quantity;
+        signup.updated_at = dateSubmitted;
+        signup.draft_reportback_submission = undefined;
```

Comment:
@sergii-tkachenko I'm specifically using `undefined` here to remove the `draft_reportback_submission` field from the Signup document, to indicate we don't have a draft Reportback Submission for this Signup.
<img width="813" alt="screen shot 2016-10-17 at 10 35 39 am" src="https://cloud.githubusercontent.com/assets/1236811/19448179/925f0c6c-9455-11e6-8592-21137b48c59e.png">

Do you think it'd be cleaner to include the `draft_reportback_submission` property, with set to `null`? (which is what the behavior would be if we pass `null`)